src/d3d11/d3d11_context.cpp

```diff
@@ -909,7 +909,7 @@ namespace dxvk {
           cDstImage->mipLevelExtent(cDstLayers.mipLevel));
       });
     } else {
-      const VkFormat format = m_parent->LookupFormat(
```

Comment:
This seems like a pointless change.

Reply:
ye, fair, it only happened because I had put the code to use the src image format here first.

Not like it really changes anything.